### DISCUSSION TITLE
How to change link text foreground?

### DISCUSSION THREAD

**Discussion by alexandrehtrb** [AvaloniaUI/AvaloniaEdit#348] [Q&A] [answered]
Hello,

How to change link text foreground? The default color, blue, works well for white background, but not on dark background.

I found this line, but it is in TextView:

https://github.com/AvaloniaUI/AvaloniaEdit/blob/870f53ad8372365f0ed84ebd15dfcea8e8ee803d/src/AvaloniaEdit/Rendering/TextView.cs#L567

**Comment by Washi1337** [comment, marked as answer]
You can access the `TextView` of the `TextEditor` control like so:

```csharp
TextEditor editor = ...
editor.TextArea.TextView.LinkTextForegroundBrush = Brushes.Cyan;
```

**Comment by alexandrehtrb** [comment]
Thanks!
